extensions/wikia/AdEngine/js/provider/gpt/googleSlots.js

```diff
@@ -0,0 +1,25 @@
+define('ext.wikia.adEngine.provider.gpt.googleSlots', function() {
```

Comment:
not sure because there is no much logic in this module however I can add it if you find it useful